indico/modules/events/abstracts/templates/management/abstract.html

```diff
@@ -1,6 +1,180 @@
+                        Track: {{ title }}
+                    {%- endtrans %}
+                </h3>
+                {% for review in reviews %}
+                    {{ _render_review(review) }}
+                {% endfor %}
+            {% endif %}
+        {% else %}
+        {% endfor %}
+    </div>
+{% endmacro %}
+
+{% macro _render_review(review) %}
+    <div class="i-timeline-item">
+        <div class="i-timeline-item-label action abstract-comment-pic"></div>
+        <div class="i-timeline-item-box header-indicator">
+            <div class="i-box-header flex">
+                <div class="stretch">
+                    {% trans name=review.user.name, date=review.created_dt|format_human_date -%}
+                        <strong>{{ name }}</strong> reviewed on {{ date }}
+                    {%- endtrans %}
+                    {% if review.modified_dt %}
+                        {{ render_edited_hint(comment.modified_dt) }}
+                    {% endif %}
+                </div>
+                <div class="abstract-comment-badges">
+                    <span>
+                        {% trans %}Proposes to{% endtrans %}
+                    </span>
+                    {{ _render_review_state(review) }}
+                    {% if review.score %}
+                        {% set score = review.score %}
+                        {# TODO: show max score #}
+                        <span class="i-tag outline highlight" title="{% trans %}Score: {{ score }}/...{% endtrans %}">
+                            {{ score }}
+                        </span>
+                    {% endif %}
+                </div>
+            </div>
+            {% if review.ratings %}
+                {% for rating in review.ratings %}
+                    <div>
+                        {# TODO: UI #}
+                        {{ rating.question.text }}
+                        {{ rating.value }}
+                    </div>
+                {% endfor %}
+            {% endif %}
+            {% if review.comment %}
+                <div class="titled-rule">
+                    {% trans %}Comment{% endtrans %}
+                </div>
+                <div class="i-box-content">
+                    {{ review.comment }}
+                </div>
+            {% endif %}
+        </div>
+    </div>
+{% endmacro %}
+
+{% macro _render_decision_box(abstract) %}
+    <div id="abstract-decision-box" class="i-timeline-item">
+        <div class="i-timeline-item-label action icon-hammer"></div>
+        <div class="i-timeline-item-box content-indicator">
+            <div class="i-box-content">
+                <div class="flex">
+                    <div class="stretch">
+                        <strong>
+                            {% trans %}Review summary{% endtrans %}
+                        </strong>
+                    </div>
+                    <div class="abstract-comment-badges">
+                        {% trans %}Reviews are{% endtrans %}
```

Comment:
`{% trans state=_render_abstract_reviewing_state(abstract) %}Reviews are {{ state }}{% endtrans %}`

Not super pretty either but less awful to translate if it's not "<Reviews are> <xxx>" in that order in some other language. If you change it make sure to test it, I don't know if you need `|safe` here. Probably not but just in case better test it